serverhttpy/server.py

```diff
@@ -70,10 +70,19 @@ def register_endpoint(self, endpoint: Endpoint):
         self.__endpoints[endpoint] = endpoint
 
     def __execute_action(self, request: Request) -> Response:
-        searched_endpoint = Endpoint(request.uri.path, request.method)
-        if searched_endpoint not in self.__endpoints:
+        potential_endpoints = []
+        searched_endpoint = None
+        for endpoint in self.__endpoints:
+            if Server.compare_paths(endpoint.path.path_elements, request.uri.path.path_elements):
+                potential_endpoints.append(endpoint)
+        if len(potential_endpoints) == 0:
             return Response(StatusCode.NotFound)
-        return self.__endpoints[searched_endpoint].execute(request)
+        elif len(potential_endpoints) > 1:
+            searched_endpoint = Server.get_correct_endpoint(potential_endpoints, request.uri.path.path_elements)
+        else:
```

Comment:
Lets throw an exception here with information about multiple identical endpoints. It could save some time our users during debugging ;)